src/main/java/net/rptools/maptool/client/functions/DrawingFunctions.java

```diff
@@ -260,19 +368,78 @@ protected JsonObject getDrawingJSONInfo(String functionName, Zone map, GUID guid
     dinfo.addProperty("isEraser", el.getPen().isEraser() ? BigDecimal.ONE : BigDecimal.ZERO);
     dinfo.addProperty("penWidth", el.getPen().getThickness());
     dinfo.add("path", pathToJSON(d));
-    if (el.getDrawable() instanceof AbstractTemplate t) {
-      dinfo.addProperty("templateSize", t.getRadius());
+    dinfo.addProperty(
+        "isTemplate",
+        el.getDrawable() instanceof AbstractTemplate ? BigDecimal.ONE : BigDecimal.ZERO);
+    if (el.getDrawable() instanceof AbstractTemplate at) {
+      JsonObject template = new JsonObject();
+      template.addProperty("radius", at.getRadius());
```

Comment:
Backwards compatibility issue here: we shouldn't change the `"templateSize"` key to `"radius"` since macros written pre-1.19 may expect to find the former.

Also, in keeping with my other comment on the new template functions, wall templates don't really have a "radius" per se, so even if we added `"radius"` in addition to `"templateSize"`, we ought to exclude that property for wall templates.